modules/service/src/main/resources/db/migration/V0890__ft_cfp_title.sql

```diff
@@ -0,0 +1,33 @@
+
+-- Format the title of the CFP.
+CREATE OR REPLACE FUNCTION format_cfp_title(
+  cfp_type  e_cfp_type,
+  semester  d_semester,
+  startDate date,
+  endDate   date,
+  instruments d_tag[]
+) RETURNS text AS $$
+DECLARE
+  instrument_list text;
+BEGIN
+  instrument_list := array_to_string(instruments, ', ');
+  RETURN CASE
+    WHEN cfp_type = 'demo_science'        THEN concat(semester, ' Demo Science')
+    WHEN cfp_type = 'directors_time'      THEN concat(semester, ' Director''s Time')
+    WHEN cfp_type = 'fast_turnaround'     THEN
+      concat(
+        left(semester, -1),
+        ' ',
+        to_char(date_trunc('month', startDate) - INTERVAL '2 months', 'FMMonth'),
```

Comment:
The real diff to the function would be this single line.  Nothing else changes.

Reply:
Is it always 2 months?